.ci/validation.cmd

```diff
@@ -32,6 +32,19 @@ if "%OPTION%" ==  "site_without_verify" (
   goto :END_CASE
 )
 
+if "%OPTION%" == "git_diff" (
+  for /f "delims=" %%a in ('git status ^| findstr /c:"Changes not staged"') do set output=%%a
```

Comment:
Just pointing out that this command relies on the english locale.

Reply:
So is there any locale compliance that I haven't complied with?

Reply:
We have a lot of scripts that rely on English locale. We can presume for now that CI will be  English only for next few years for sure

Reply:
Ok. Let's merge then.